sdk/search/search-documents/test/utils/setupIndex.ts

```diff
@@ -415,3 +419,275 @@ export async function populateIndex(client: SearchClient<Hotel>): Promise<void>
     count = await client.getDocumentsCount();
   }
 }
+
+// eslint-disable-next-line @azure/azure-sdk/ts-use-interface-parameters
+export async function createDataSourceConnections(client: SearchIndexerClient): Promise<void> {
+  const connectionString:string = "AccountEndpoint=https://hotels-docbb.documents.azure.com:443/;AccountKey=4UPsNZyFAjgZ1tzHPGZaxS09XcwLrIawbXBWk6IixcxJoSePTcjBn0mi53XiKWu8MaUgowUhIovOv7kjksqAug==;Database=SampleData";
+  const dataSourceConnections:SearchIndexerDataSourceConnection[] = [{
+    name: "my-data-source-1",
```

Comment:
nit: can we build this array programmatically instead of repeating it out? It seems very easy to replace with a for loop that repeats 5 times.

Reply:
Done